src/Middleware/ApiAuthorizedOnlyMiddleware.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+namespace kissj\Middleware;
+
+use kissj\Event\EventRepository;
+use kissj\FlashMessages\FlashMessagesInterface;
+use kissj\User\UserController;
+use Psr\Http\Message\ResponseInterface as Response;
+use Psr\Http\Message\ServerRequestInterface as Request;
+use Psr\Http\Server\RequestHandlerInterface as ResponseHandler;
+use Symfony\Contracts\Translation\TranslatorInterface;
+
+class ApiAuthorizedOnlyMiddleware extends BaseMiddleware
+{
+    public function __construct(
+        private readonly EventRepository        $eventRepository
+    )
+    {
+    }
+
+    public function process(Request $request, ResponseHandler $handler): Response
+    {
+        $secret = $request->getHeader('Authorization')[0];
+
+        if (!$secret) {
```

Comment:
je lepší vyjádřit, co se tímto myslí přesněji za typ - je to `=== null`, nebo `is_string()`, nebo ještě něco jinéhoM

Reply:
OK - co značí, že tam ten header není? Jestli `null`, kontroluj to proti tomu. Jestli `[]`, tak proti tomu atp.